test/win-bats/collection.xml

```diff
@@ -1971,7 +1971,7 @@
     call :verify_line 15 x "Formatting Report..."
 	</case>
 
-	<case name="x:variable should be evaluated only once (XSLT)">
+	<case ifdef="SAXON12_INITIAL_ISSUES_FIXED" name="x:variable should be evaluated only once (XSLT)">
```

Comment:
Agreed that it's an implementation-specific zone.

In #1752, I updated the test to find only the `* [1]: xs:string: value-` lines. (d7fa240395dbae3886c558d2ee5bd98a97fcd73d)
I changed the test from CLI to Ant just because Ant `-logfile` option was handy.

Reply:
Your approach in #1752 looks good.